src/executables/util_cs.c

```diff
@@ -3993,3 +3993,311 @@ tde (UTIL_FUNCTION_ARG * arg)
 error_exit:
   return EXIT_FAILURE;
 }
+
+static time_t
+parse_date_string_to_time (char *date_string)
```

Comment:
Why don't you use the existing standard solutions? https://stackoverflow.com/questions/11213326/how-to-convert-a-string-variable-containing-time-to-time-t-type-in-c

Reply:
Oh.. Thank you. I will take a look at what you sent.

Reply:
I replace it with ssanf function.

There are two  candidate functions for parsing date format string and converting it to time_t. 
1. std::get_time
   - Can not handle a exception if some parts(day, month, year, hour, minutes or seconds) are not input 
2. sscanf 
   - Have to check that the parts are set correctly, but can handle the exception mentioned above.